app/actors/iiif_print/actors/iiif_print_upload_actor.rb

```diff
@@ -4,18 +4,20 @@ class IiifPrintUploadActor < Hyrax::Actors::AbstractActor
       # An actor which locates all uploaded PDF paths and
       # spins off IiifPrint::CreatePagesJob to split them.
       def create(env)
-        # TODO: test what happens when ensure_title is removed.
         ensure_title(env)
         @pdf_paths = []
-        hold_upload_paths(env) if responds_to_split?(env.curation_concern)
+        @prior_pdfs_count = 0
+        hold_upload_paths(env) if iiif_print?(env)
         next_actor.create(env) && after_other_actors(env)
       end
 
       def update(env)
-        # TODO: test what happens when ensure_title is removed.
         ensure_title(env)
         @pdf_paths = []
-        hold_upload_paths(env) if responds_to_split?(env.curation_concern)
+        if iiif_print?(env)
+          hold_upload_paths(env)
+          count_existing_pdfs(env)
```

Comment:
Is it possible to not pass in `env`?  It doesn't look like it's being used later so I'm wondering how come it's even needed?

Reply:
I assume you are referring to `count_existing_pdfs` since everything else needs env? That method is a todo for now, and WILL need the env once it's implemented, so that's why I put it in.